### DISCUSSION TITLE
gamify the achievement gain by showing all locked items and include leaderboards

### DISCUSSION THREAD

**Discussion by junguler** [community/community#21066] [Other Feature Feedback, Questions, & Ideas] [answered]
don't make us hunt for a certain achievement and how to unlock it, list them all in a page and give clear info about how to unlock them

the leaderboard is so obvious, put users in groups divided by follower count, region, language and other qualifiers and include them in local and global leaderboards

**Comment by kurnakovv** [comment, marked as answer]
@junguler It's a good idea, but for now, we can use this [repository](https://github.com/Schweinepriester/github-profile-achievements) to understand what achievements are, how to get them and who completed them 100%

If I helped you, can you mark this answer?

**Reply by junguler** [reply]
thanks for pointing me out to that page, i'll bookmark this page and if i haven't gotten any further replies i'll mark your comment as the answer

**Comment by jfullstackdev** [comment]
we have different opinions to this, as for me, the idea is that, it's always given as a surprise, because the main aim is to continue contributing to open source not because of the badges, because you want to contribute, 

as we all see what's happening after the release, most are just asking how to achieve this, because the aim is not to contribute anymore but to simply have it,

**Reply by junguler** [reply]
i understand your reasoning but like you said most people are interested in the game aspect of it so why not embrace what the majority want?

**Reply by jfullstackdev** [reply]
I'm not a member of GitHub Staff, I'm just commenting here

maybe this can be implemented, what I was saying is that, as for me, first, it's surprising if it's given by surprise, as we can see by the multitude of posts after badges are released, second, again this is just my opinion, is not for the sake of the badges but just a recognition of our efforts contributing to open source, that's why it's best to give them as a surprise

and someone created the guide for us

https://github.com/Schweinepriester/github-profile-achievements

**Reply by junguler** [reply]
i get your point and it's a valid and reasonable one to have but unfortunately people have become dopamine addicts and can't help themselves
